kiwi/defaults.py

```diff
@@ -61,6 +61,12 @@ def __init__(self):
         ]
 
     @classmethod
+    def is_obs_worker(self):
+        # the presence of /.buildenv on the build host indicates
+        # we are building inside of the open buildservice
+        return os.path.exists('/.buildenv')
+
```

Comment:
Wouldn't it be better to test it against the usage of `suse://` schema in the uri? This way all the ad-hoc code relies on the same entry point, and we have only the path of the repo hardcoded (even `.buildenv` doesn't look like is going to change).

However I have to admit that I do not have a clear idea where we could store the `is_obs_worker` flag. Maybe in xml_state when parsing the uri?